client/src/extension.ts

```diff
@@ -117,22 +116,25 @@ export function activate(context: ExtensionContext) {
         return;
     }
     /*
-    * List of all the yseopml file extensions known by this extension as set in `client/package.json`.
-    */
-    const yseopmlExtensions = [
-        'kao',
-        'yclass',
-        'yobject',
-        'ycomplete',
-        'dcl',
-        'yml',
-    ];
+     * List of all the yseopml file extensions known by this extension as set in `client/package.json`.
```

Comment:
Would be nice to programmatically retrieve this list from the `client/package.json`. Did you check that there’s no way to access it from here?

Reply:
I didn't find a way.

Reply:
:(